internal/lib/constants.go

```diff
@@ -196,6 +196,8 @@ const (
 	Namespace                                  = "Namespace"
 	VrfContextNotFoundError                    = "VrfContext not found"
 	K8s_1_28                                   = "1.28"
+	HTTPMethodGet                              = "GET"
```

Comment:
can we use method constants from net.http package .

Reply:
I will take care this in new PR.